builder/proxmox/iso/config.go

```diff
@@ -32,8 +32,11 @@ type Config struct {
 	ISODownloadPVE bool `mapstructure:"iso_download_pve"`
 	// If true, remove the mounted ISO from the template
 	// after finishing. Defaults to `false`.
-	UnmountISO      bool `mapstructure:"unmount_iso"`
-	shouldUploadISO bool
+	UnmountISO bool `mapstructure:"unmount_iso"`
+	// Keep CDRom device attached to template if unmounting ISO. Defaults to `false`.
+	// Has no effect if unmount is `false`
+	UnmountKeepDevice bool `mapstructure:"unmount_keep_device"`
```

Comment:
The additional_iso_files config is in common, and the proxmox-iso builder's iso device is in iso, so the keep option has been added in both places.

Reply:
OK I didn't understand that the option was to point at different devices.
So just to be clear, is the `additionalISOConfig` squashed into the config, or is it imported as a block for the ISO builder? If it is squashed, we'll have a conflict between the two options.

EDIT: looked at the code, looks like it'll be fine since the `additionalISOConfig` objects are added as a slice to the configuration, so each object will not be squashed into the config directly, both options won't conflict! We can keep them both, looks legit to me